homeassistant/helpers/location.py

```diff
@@ -41,3 +48,61 @@ def closest(
             longitude,
         ),
     )
+
+
+def coordinates(
```

Comment:
Can we rename this function to be something like "find_coordinates" ? Just `coordinates` seems a little basic.

Reply:
Renamed it to find_coordinates.